wrapper/wrapper.go

```diff
@@ -10,15 +10,26 @@ import (
 	"os/signal"
 	"syscall"
 	"time"
+	"encoding/json"
 
 	"github.com/domsolutions/gopayloader/config"
 	"github.com/domsolutions/gopayloader/pkgs/payloader"
 )
 
-func RunGoPayLoader(reqURI, mTLScert, mTLSKey string, disableKeepAlive bool, reqs int64, conns uint, totalTime time.Duration, skipVerify bool, readTimeout, writeTimeout time.Duration, method string, verbose bool, ticker time.Duration, jwtKID, jwtKey, jwtSub, jwtIss, jwtAud, jwtHeader string, headers []string, body, bodyFile string, client string) error {
+// Converts jwtCustomClaimsJSON from string to map[string]string
+func JwtCustomClaimsJSONStringToMap(jwtCustomClaimsJSON string) map[string]string {
+	jwtCustomClaimsMap := map[string]string{}
+
+  json.Unmarshal([]byte(jwtCustomClaimsJSON), &jwtCustomClaimsMap)
```

Comment:
Can you check the error here? it's been ignored

Reply:
Thanks! Made the change to return the errors and actually found out there was an error with bools and other types not being able to be converted properly so I fixed that bug as well